api/common/common.go

```diff
@@ -34,3 +38,12 @@ func ResponsePack(errCode ErrCode) map[string]interface{} {
 	}
 	return resp
 }
+
+func ReturnCert(h *tls.ClientHelloInfo) (*tls.Certificate, error) {
+	cert, err := tls.LoadX509KeyPair(config.Parameters.HttpsJsonCert, config.Parameters.HttpsJsonKey)
```

Comment:
How about load KeyPair specified by tls.ClientHelloInfo.ServerName in order to support SNI (RFC 4366)?